internal/commands/run.go

```diff
@@ -43,6 +46,9 @@ func run(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
+	if !verbose {
```

Comment:
These two things (verbosity and interactivity) seem to be orthogonal: you can have both interactive and verbose output at the same time.

Take a look at `-1` argument of the [`ls`](https://man.openbsd.org/ls) command if you want an option to disable interactive output.

Reply:
I don't see how it can play well with the incremental redrawing. I can try to add log levels to echelon and have description only show messages for a particular log level. Then we can try to replace logrus completely with echelon. 🤔 What do you think?

Reply:
That's an interesting idea. Could the echelon library provide a logrus-compatible interface to simplify integrations?

This way the only thing we'll have to do here is to swap the logger.

Reply:
You still need to create the hierarchy but the `Debugf` and context things can be replicated. Will experiments with it. 👍